backend/de.metas.serviceprovider/src/main/java/de/metas/serviceprovider/issue/IssueRepository.java

```diff
@@ -100,28 +93,41 @@ public IssueEntity getById(@NonNull final IssueId issueId, final boolean loadLab
 					.setParameter("S_Issue_Id", issueId);
 		}
 
-		return buildIssueEntity(record, loadLabels);
+		return buildIssueEntity(record);
 	}
 
 	@NonNull
-	public Optional<IssueEntity> getByIdOptional(@NonNull final IssueId issueId, final boolean loadLabels)
+	public Optional<IssueEntity> getByIdOptional(@NonNull final IssueId issueId)
 	{
 		final I_S_Issue record = getRecordOrNull(issueId);
 
 		return record != null
-				? Optional.of(buildIssueEntity(record, loadLabels))
+				? Optional.of(buildIssueEntity(record))
 				: Optional.empty();
 	}
 
 	@NonNull
-	public Optional<IssueEntity> getByExternalURLOptional(@NonNull final String externalURL, final boolean loadLabels)
+	public Optional<IssueEntity> getByExternalURLOptional(@NonNull final String externalURL)
 	{
 		return queryBL.createQueryBuilder(I_S_Issue.class)
 				.addOnlyActiveRecordsFilter()
 				.addEqualsFilter(I_S_Issue.COLUMNNAME_IssueURL, externalURL)
 				.create()
 				.firstOnlyOptional(I_S_Issue.class)
-				.map(record -> buildIssueEntity(record, loadLabels));
+				.map(this::buildIssueEntity);
+	}
+
+	@NonNull
+	public ImmutableList<IssueEntity> getDirectlyLinkedSubIssues(@NonNull final IssueId parentIssueId)
+	{
+		return queryBL.createQueryBuilder(I_S_Issue.class)
+				.addOnlyActiveRecordsFilter()
+				.addEqualsFilter(I_S_Issue.COLUMNNAME_S_Parent_Issue_ID, parentIssueId.getRepoId())
```

Comment:
```suggestion
				.addEqualsFilter(I_S_Issue.COLUMNNAME_S_Parent_Issue_ID, parentIssueId)
```
it will figure out that parentIssueId is a RepoIdAware